azkaban-common/src/main/java/azkaban/executor/FlowStatusChangeEventListener.java

```diff
@@ -76,6 +76,7 @@ public FlowStatusChangeEventListener(final Props props) {
     metaData.put(EventReporterConstants.SUBMIT_TIME, String.valueOf(flow.getSubmitTime()));
     metaData.put(EventReporterConstants.FLOW_VERSION, String.valueOf(flow.getAzkabanFlowVersion()));
     metaData.put(EventReporterConstants.FLOW_STATUS, flow.getStatus().name());
+    metaData.put(EventReporterConstants.EXECUTION_SOURCE, flow.getExecutionSource());
```

Comment:
There's a flowrunnereventlistner. I'm not sure if you can get executionSource of the flow here. But it should be available in the running stage. Please verify.

Reply:
Made the change to add this information to the FLOW_STARTED event's metadata in FlowRunnerEventListener.